recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
take a look at fedora packages, they use make %{smpflags}, with smpflags set in the environment, here conda does not set any MAKEFLAGS for us, so we have to specify our own.